components/data-table/index.jsx

```diff
@@ -516,7 +516,7 @@ class DataTable extends React.Component {
 			component = (
 				<div
 					className="slds-table_header-fixed_container"
-					style={{ height: '100%' }}
+					style={{ borderTop: '1px solid #dddbda', height: '100%' }}
```

Comment:
Please use design token $color-gray-5
https://github.com/salesforce/design-system-react/blob/master/docs/codebase-overview.md#use-design-tokens-for-inline-styles

Example: https://github.com/salesforce/design-system-react/blob/master/components/global-navigation-bar/dropdown-trigger.jsx#L145